src/harness/tests/testdata/registration/reg2_a_singlestep.json

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
Question: do you only want to use on sigle file, that is, only 'reg1_a_singlestep.json', and when required modify the values in memory during the test case execution? is that the idea behind this?